src/components/GalleryEditor/GalleryEditor.tsx

```diff
@@ -46,37 +38,6 @@ export function GalleryEditor({ queryRef }: GalleryEditorProps) {
     queryRef
   );
 
-  const canGoBack = useCanGoBack();
```

Comment:
all this stuff belongs in the parent component now (edit gallery, onboarding edit gallery)